src/msg/Message.cc

```diff
@@ -326,7 +326,10 @@ Message *decode_message(CephContext *cct,
     if (front_crc != footer.front_crc) {
       if (cct) {
 	ldout(cct, 0) << "bad crc in front " << front_crc << " != exp " << footer.front_crc
-		      << " from " << conn->get_peer_addr() << dendl;
+	              << dendl;
+#ifndef WITH_SEASTAR
+	ldout(cct, 0) << " from " << conn->get_peer_addr() << dendl;
```

Comment:
This splits the single debug into two separated ones. I would prefer to keep it untouched in the classical.

Reply:
Classic should remain untouched. I wanted for Crimson to also be able to print log message when verifying the crc (in this case). Since `conn->get_peer_addr()` is not possible in Crimson I have separated this part into an `ifndef` block. Same goes for the other two instances of crc verifying.
Do you agree with this approach?

Reply:
Is it the preferable way?
```c++
ldout(cct, 0) << "bad crc in front " << front_crc << " != exp " << footer.front_crc
#ifndef WITH_SEASTAR
              << " from " << conn->get_peer_addr()
#endif
              << dend;
```

Reply:
> Is it the preferable way?
 
LGTM, updated.